pkgs/tools/X11/x11spice/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitLab, autoreconfHook, pkgconfig
+, xorg, gtk2, spice, spice-protocol
+}:
+
+stdenv.mkDerivation rec {
+  pname   = "x11spice";
+  version = "2019-08-20";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.freedesktop.org";
+    owner  = "spice";
+    repo   = "x11spice";
+    rev    = "51d2a8ba3813469264959bb3ba2fc6fe08097be6";
+    sha256 = "0va5ix14vnqch59gq8wvrhw6q0w0n27sy70xx5kvfj2cl0h1xpg8";
```

Comment:
I'm personally not a big fan of this indentation style, it either creates unnecessary diff noise when adding a new attribute which is longer. If we now have something to automatically format things like this, I would be happy for a heads up. If you really prefer this style of indent I would also let it pass

Reply:
No, it's nothing automatic: I have been formatting like this since I started contributing in 2015, which is a habit I got from writing haskell.
Since you are the second reviewer that recently disliked it I'm just going to drop it to. I'm just surprised that no one ever mentioned this before (I did around 300 PRs) if it's not generally accepted.

Reply:
Done.